src/iOS/Xamarin.iOS/ViewControllers/SettingsViewController.cs

```diff
@@ -0,0 +1,452 @@
+            _licensesView.NavigationDelegate = new BrowserLinksNavigationDelegate();
+
+            // View for managing offline data for samples.
+            _downloadView = new UIStackView { BackgroundColor = UIColor.White, Hidden = true };
+            _downloadView.Axis = UILayoutConstraintAxis.Vertical;
+            _downloadView.LayoutMarginsRelativeArrangement = true;
+            _downloadView.Alignment = UIStackViewAlignment.Fill;
+            _downloadView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            // Label for download status.
+            _downloadLabel = new UILabel() { Text = "Ready" };
+
+            // Buttons for downloading or deleting all items.
+            _buttonToolbar = new UIToolbar();
+
+            _downloadAllButton = new UIBarButtonItem() { Title = "Download all" };
+            _deleteAllButton = new UIBarButtonItem() { Title = "Delete all" };
+            _cancelButton = new UIBarButtonItem() { Title = "Cancel" };
+
+            _buttonToolbar.Items = new[]
+            {
+                _downloadAllButton,
+                new UIBarButtonItem(UIBarButtonSystemItem.FlexibleSpace),
+                _deleteAllButton,
+            };
+
+            // Table of samples with downloadable items.
+            _downloadTable = new UITableView();
+            _downloadTable.Source = new SamplesTableSource(_samples, _downloadLabel);
+            _downloadTable.RowHeight = 50;
+            _downloadTable.AllowsSelection = false;
+
+            // Add the views to the download view.
+            _downloadView.AddArrangedSubview(_downloadLabel);
+            _downloadView.AddArrangedSubview(_downloadTable);
+            _downloadView.AddArrangedSubview(_buttonToolbar);
+
+            // Add sub views to main view.
+            View.AddSubviews(_aboutView, _licensesView, _downloadView);
+
+            NSLayoutConstraint.ActivateConstraints(new[]
+            {
+                 _aboutView.TopAnchor.ConstraintEqualTo(View.TopAnchor),
+                 _aboutView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                 _aboutView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                 _aboutView.BottomAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.BottomAnchor),
+
+                 _licensesView.TopAnchor.ConstraintEqualTo(View.TopAnchor),
+                 _licensesView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                 _licensesView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                 _licensesView.BottomAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.BottomAnchor),
+
+                 _downloadView.TopAnchor.ConstraintEqualTo(View.TopAnchor),
+                 _downloadView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                 _downloadView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                 _downloadView.BottomAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.BottomAnchor),
```

Comment:
I recommend adding the toolbar directly to the view, constraining toolbar bottom to `SafeAreaLayoutGuide.BottomAnchor`, and using `_downloadView.BottomAnchor.ConstraintEqualTo(_buttonToolbar.TopAnchor)` or similar. Otherwise the toolbar doesn't look right on devices with a notch.

Reply:
I recommend testing any changes on an iPhone 11 in both landscape and portrait to be sure things look right; currently the downloads page isn't making good use of the left, right, or bottom sides in landscape.

Reply:
I can check this out, though I don't have access to an iPhone 11, would an iPhone X suffice?

Reply:
Any variant of iPhone X or iPhone 11 will show the issue

Reply:
I added leading and trailing contraints to  the toolbar and it fixed the issue.